src/main/java/core/basesyntax/service/impl/FruitTransactionProcessorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationHandlerStrategy;
+import core.basesyntax.strategy.OperationHandlerStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class FruitTransactionProcessorImpl implements TransactionProcessor {
+    @Override
+    public void process(List<FruitTransaction> fruitTransactionsList,
```

Comment:
```suggestion
    public void process(List<FruitTransaction> fruitTransactions,
```
imo we usually don't include type in variable's name, java is statically typed anyway